extension/syntaxes/templateSettingsSyntax.json

```diff
@@ -222,9 +222,49 @@
       "type": "boolean",
       "examples": [],
       "default": true
+    },
+    "postConversionTasks": {
+      "$id": "#root/postConversionTasks",
+      "title": "Post Conversion Tasks",
+      "description": "Specifies commands to be run after conversion is completed. Commands are run in order of definition.",
+      "default": [],
+      "type": "array",
+      "items": {
+        "$ref": "#/definitions/postConversionTaskItem"
+      }
     }
   },
   "definitions": {
+    "postConversionTaskItem": {
+      "description": "Defines a command to be executed by VSCode",
+      "type": "object",
+      "uniqueItems": true,
```

Comment:
Honestly, I don't know. Maybe start out like this and change it on request? I have no strong opinion on this so you can decide :smile:

Reply:
Start like this 👍
But your test data would have failed if it was done manually! 😜

Reply:
Haha I was waiting for that comment :p 

Yes, let's remove it. The only argument to keep it would be to try and help users not accidentally duplicate tasks. But in this case I think it's better to allow than to restrict. If someone succeeds in turning this into a foot gun... well :clap:

Reply:
removed